data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1020 @@
+			label approach
+			choice
+				`	(Approach the location.)`
+					goto encounter
+				`	"You know a lot about tracking for a scholar."`
+			`	"Ah," Sieasej says, pausing for a moment. "A product of some training I've not used in many years. Perhaps Sioeora is as forward-thinking as they would us believe."`
+			label encounter
+			`	You make your way along one of the walkways to the west, brushing aside the enormous leaves of the overgrowing plants. Sieasej continues to wave their triangulation device in the air and quietly relays your progress; as you get closer, you hear a gentle scuffling that you realize precedes every detection by the device. After making your way through one particularly-dense wall of flora, you spot your target: a glassy-winged animal perched in a plant over the walkway's edge, probing the branch it stands on with long feelers extending from its head.`
+			choice
+				`	(Try to tranquilize it.)`
+					goto tranq
+					to display
+						or
+							has "outfit: Enforcer Riot Staff"
+							has "outfit: Pug Peacekeeping Staff"
+				`	(Try to catch it with a net.)`
+					goto net
+				`	(Stay still.)`
+					goto lure-1
+			label tranq
+			branch tranqed
+				random < 70
+			label tranqed-alarmed
+			`	You take aim and fire a shot from the <tranq h2h> at the animal, but it darts away at the last moment so that the blast only just grazes its body. It plummets to the ground shrieking, and while it soon goes quiet as the tranquilizer takes effect, the shriek is echoed by other animals in the area. Soon after those quiet as well, your ears are pierced by a loud, warbling wail from the distance.`
+				goto escape
+			label tranqed
+			`	You take aim and fire a shot from the <tranq h2h> at the center of the animal. It briefly jumps to escape but soon plummets to the ground in a puff of leaves and dusty feathers.`
+				goto bring-tranq
+			label net
+			`	You toss a net over the animal and it falls to the ground, where it shrieks for a duration before going quiet. The shriek is soon echoed by similar ones from other animals in the area, and once those go quiet as well, your ears are pierced by a loud, warbling wail in the distance.`
+				goto escape
+			label lure-1
+			branch lure-1-success
+				( random + has "Pookie, Part 5: offered" * 2 + has "Remnant: Face to Maw 1: completed" * 2 ) < 95
+			`	The animal remains stationary for a moment before darting away.`
+				goto lured-failed
```

Comment:
Everything else is `lure-something`, you might want to change this label as well.